docs/content/theme/color-palette.md

```diff
@@ -18,7 +18,12 @@ tags = [ "theme" ]
 
 ## Text & links
 
-Text and links are the most fundamental pieces of any web property, so they have their own variables aside from other UI elements. They also respond to context, such as a bit of text in a dark gray card would need to change to white. Or a link in a dark gray card would need to become a lighter shade of blue.  Here are a few examples:
+Text and links, being fundamental pieces of any web property, have their own set of variables. 
```

Comment:
TODO: We can remove broadcast text in the future and just have broadcast variables for links

Reply:
We can address this in a separate PR: https://github.com/patternfly/patternfly-elements/issues/1009